plasma_framework/python_tests/tests/tests_utils/plasma_framework.py

```diff
@@ -149,8 +169,19 @@ def startStandardExit(self, utxo_pos, output_tx, output_tx_inclusion_proof, **kw
         args = (utxo_pos, output_tx, b'', output_tx_inclusion_proof)
         return self.payment_exit_game.startStandardExit(args, **kwargs)
 
-    def challengeStandardExit(self, standard_exit_id, challenge_tx, input_index, challenge_tx_sig):
-        raise NotImplementedError
+    def challengeStandardExit(self, standard_exit_id, challenge_tx, input_index, challenge_tx_sig, exiting_tx,
+                              **kwargs):
+        """NOTICE: takes more arguments than the original one"""
```

Comment:
It took me a while to understand this `NOTICE`. Might be better to be clear it's ALD vs Rootchain.sol or probably comment that extra args for potential future feature which is non-used now.